_i18n/de/resources/user-guides/mine-to-pool.md

```diff
@@ -151,94 +127,70 @@ After these modifications, your configuration should look something like this:
-4. Select the User Rights Assignment folder.
-5. The policies will be displayed in the details pane.
-6. In the pane, double-click Lock pages in memory.
-7. In the Local Security Setting – Lock pages in memory dialog box, click Add User or Group.
-8. In the Select Users, Service Accounts, or Groups dialog box, add an account that you will run the miner on
-9. Reboot for change to take effect.
+>1. Klicken Sie im Menü "Start" auf "Ausführen". Geben Sie "gpedit.msc" im Feld "Öffnen" ein.
+2. Erweitern Sie in der Konsole "Editor für lokale Gruppenrichtlinien" die Option "Computerkonfiguration" und dann "Windows-Einstellungen".
+3. Erweitern Sie "Sicherheitseinstellungen" und dann "Lokale Richtlinien".
+4. Wählen Sie den Ordner "Zuweisen von Benutzerrechten" aus. 
+Die Richtlinien werden im Detailbereich angezeigt.
+5. Doppelklicken Sie im Detailbereich auf "Sperren von Seiten im Speicher".
+6. Klicken Sie im Dialogfeld "Lokale Sicherheitseinstellung" > "Sperren von Seiten im Speicher auf Benutzer oder Gruppe hinzufügen".
+7. Wählen Sie im Dialogfeld "Select Users, Service Accounts, or Groups" (Benutzer, Dienstkonten oder Gruppen auswählen) das SQL Server-Dienstkonto aus.
+8. Starten Sie SQL Server neu, damit diese Einstellung übernommen wird.
 
-You may also need to launch the miner as administrator.
+Es kann zudem sein, dass du den Miner als Administrator starten musst.
 
-#### Allowing large pages on Linux
+#### Große Seiten auf Linux erlauben
+
+Beende zunächst den Miner (falls dieser laufen sollte) und führe die folgenden Befehle aus, um große Seiten zu erlauben; danach starte den Miner via Root:
 
-Firstly stop the miner (if it's running), run the following commands to enable
-large pages and then start the miner as root:
 
 	sudo sysctl -w vm.nr_hugepages=1168
 	sudo ./xmrig
 
-You may have to increase 1168 depending on how many NUMA nodes your
-CPU(s) have.
+Es kann sein, dass du 1168 erhöhen musst; dies ist abhängig von der Anzahl an NUMA-Nodes deines (deiner) CPUs.
 
-#### Allowing large pages on macOS
+#### Große Seiten auf macOS erlauben
 
-Huge pages are not supported on macOS.
+Große Seiten werden auf macOS nicht unterstützt.
 
-### Balance Not Increasing
+### Guthaben erhöht sich nicht
 
-Most pools are <abbr title="Pay Per Last N Shares">PPLNS</abbr> pools,
-which means that you only get paid when a miner on the pool finds a
-block. If the pool you are mining on is small, this can take a few
-days to weeks.
+Die meisten Pools sind <abbr title="Pay Per Last N Shares">PPLNS</abbr>-Pools. Dies bedeutet, dass du nur bezahlt wirst, wenn ein Miner im Pool einen Block findet. Wenn der Pool, innerhalb dessen du minest, klein ist, kann dies einige Tage bis hin zu Wochen dauern.
 
-Additionally, any blocks found must mature before they can be paid
-out. This takes 60 blocks (approx. 2 hours).
+Außerdem müssen Blöcke voll entwickelt sein, bevor sie ausbezahlt werden. Dafür werden 60 Blöcke (ungefähr zwei Stunden) benötigt.
```

Comment:
```suggestion
Außerdem müssen Blöcke mehrfach bestätigt werden, bevor sie ausbezahlt werden. Dafür werden 60 nachfolgende Blöcke (ungefähr zwei Stunden) benötigt.
```

I know this deviates a bit from the English version, but I couldn't find a better way to translate "blocks that need to mature".. wouldn't use "voll entwickelt" :)